stdlib/trace.pyi

```diff
@@ -27,7 +33,21 @@ class CoverageResults:
     ) -> tuple[int, int]: ...
     def is_ignored_filename(self, filename: str) -> bool: ...  # undocumented
 
+class _Ignore:
+    def __init__(self, modules: Iterable[str] | None = None, dirs: Iterable[StrPath] | None = None) -> None: ...
+    def names(self, filename: str, modulename: str) -> int: ...
+
 class Trace:
+    inifile: StrPath | None
+    outfile: StrPath | None
+    ignore: _Ignore
+    counts: dict[str, int]
+    pathtobasename: dict[Incomplete, Incomplete]
+    donothing: int
+    trace: int
+    start_time: int | None
+    globaltrace: Callable[[types.FrameType, str, Any], TraceFunction]
+    localtrace: Callable[[types.FrameType, str, Any], TraceFunction]
```

Comment:
Should those be `TraceFunction` directly?
The main difference is the `TraceFunction | None` return type.

`globaltrace` is passed directly to `threading.settrace` and `sys.settrace` which are both typed to accept a `TraceFunction`

`self.globaltrace` is also assigned a bunch of different possible functions that always return `None` (like `self.globaltrace_trackcallers`, `globaltrace_countfuncs`, etc.)

Reply:
Good catch. Yep, I think these should both just be `TraceFunction` directly. They're both just used for tracing, so, even if we _could_ be more precise with `self.localstrace`, it's probably fine to simplify here (and the alias gives a better sense of the intended purpose, anyway!)